src/epdiy.c

```diff
@@ -532,3 +534,12 @@ int epd_width() {
 int epd_height() {
     return display->height;
 }
+
+void epd_set_lcd_pixel_clock_MHz(int frequency) {
+#ifdef RENDER_METHOD_LCD
+    void epd_lcd_set_pixel_clock_MHz(int frequency);
```

Comment:
This line is not defined already in the header?
Or maybe it’s me that I’m too sleepy and should already stop checking code :P

Reply:
It's not quite the same, `epd_lcd_set` vs `epd_set_lcd` ;) One is the internal LCD method, for which I didn't want to include the header so I reproduced the function declaration.